src/js/smooch.jsx

```diff
@@ -96,6 +96,8 @@ export class Smooch extends Observable {
             renderLink();
             this.trigger('ready');
             return Promise.resolve();
+        }else if (/PhantomJS/.test(navigator.userAgent) && !window.SmoochTest) {
```

Comment:
we could use instead `process.env.ENV !== 'test' and inject that value with webpack when we are doing tests.

Also, I was thinking about just adding the PhantomJS string to the regex for the link and that's it.

But yeah, we can just render nothing too.

(and formatting...)